pkg/kubelet/nodestatus/setters_test.go

```diff
@@ -426,7 +439,7 @@ func TestNodeAddress(t *testing.T) {
 			shouldError: false,
 		},
 		{
-			name:              "External cloud provider gets nodeIP annotation",
+			name:              "External cloud provider gets nodeIP annotation and initializeNodeAddressesCloudProviderExternal set",
```

Comment:
```suggestion
			name:              "External cloud provider with initializeNodeAddressesCloudProviderExternal set gets nodeIP annotation",
```

Otherwise it parses as "External cloud provider gets (1) nodeIP annotation and (2) initializeNodeAddressesCloudProviderExternal set"